### PR TITLE
Update baselines

### PR DESCRIPTION
**Pull request by yuit**
Update baselines

### CODE REVIEW

File: tests/baselines/reference/project/baseline3/amd/nestedModule.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
     "use strict";
-    var outer;
     (function (outer) {
         var inner;
         (function (inner) {
             var local = 1;
             inner.a = local;
         })(inner = outer.inner || (outer.inner = {}));
-    })(outer = exports.outer || (exports.outer = {}));
+    })(exports.outer || (exports.outer = {}));
```
Comment (vladima) [inline tests/baselines/reference/project/baseline3/amd/nestedModule.js]: this change is the emit is unnecessary (given that emit for non-exported modules still matches the old emitter) , I'll keep it from merging for now

Comment (rbuckton) [inline tests/baselines/reference/project/baseline3/amd/nestedModule.js]: Yes, don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/baseline3/node/nestedModule.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-var outer;
 (function (outer) {
     var inner;
     (function (inner) {
         var local = 1;
         inner.a = local;
     })(inner = outer.inner || (outer.inner = {}));
-})(outer = exports.outer || (exports.outer = {}));
+})(exports.outer || (exports.outer = {}));
+var outer = exports.outer;
```
Comment (rbuckton) [inline tests/baselines/reference/project/baseline3/node/nestedModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsImportedInPrivate/amd/useModule.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
     "use strict";
-    var usePrivate_m4_m1;
     (function (usePrivate_m4_m1) {
         var x3 = private_m4.x;
         var d3 = private_m4.d;
         var f3 = private_m4.foo();
-    })(usePrivate_m4_m1 = exports.usePrivate_m4_m1 || (exports.usePrivate_m4_m1 = {}));
+    })(exports.usePrivate_m4_m1 || (exports.usePrivate_m4_m1 = {}));
+    var usePrivate_m4_m1 = exports.usePrivate_m4_m1;
 });
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsImportedInPrivate/amd/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsImportedInPrivate/node/useModule.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 // only used privately no need to emit
 var private_m4 = require("private_m4");
-var usePrivate_m4_m1;
 (function (usePrivate_m4_m1) {
     var x3 = private_m4.x;
     var d3 = private_m4.d;
     var f3 = private_m4.foo();
-})(usePrivate_m4_m1 = exports.usePrivate_m4_m1 || (exports.usePrivate_m4_m1 = {}));
+})(exports.usePrivate_m4_m1 || (exports.usePrivate_m4_m1 = {}));
+var usePrivate_m4_m1 = exports.usePrivate_m4_m1;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsImportedInPrivate/node/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsMultipleTimesImport/amd/useModule.js [hunk 5]
```diff
@@ -11,7 +10,8 @@ define(["require", "exports", "m4", "m4"], function (require, exports, m4, multi
         var x3 = m4.x;
         var d3 = m4.d;
         var f3 = m4.foo();
-    })(m1 = exports.m1 || (exports.m1 = {}));
+    })(exports.m1 || (exports.m1 = {}));
+    var m1 = exports.m1;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsMultipleTimesImport/amd/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsMultipleTimesImport/node/useModule.js [hunk 6]
```diff
@@ -11,7 +10,8 @@ var m1;
     var x3 = m4.x;
     var d3 = m4.d;
     var f3 = m4.foo();
-})(m1 = exports.m1 || (exports.m1 = {}));
+})(exports.m1 || (exports.m1 = {}));
+var m1 = exports.m1;
 // Do not emit multiple used import statements
 var multiImport_m4 = require("m4"); // Emit used
 exports.useMultiImport_m4_x4 = multiImport_m4.x;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsMultipleTimesImport/node/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsMultipleTimesMultipleImport/amd/useModule.js [hunk 7]
```diff
@@ -11,6 +10,7 @@ define(["require", "exports", "m4", "m5"], function (require, exports, m4, m5) {
         var x3 = m4.x;
         var d3 = m4.d;
         var f3 = m4.foo();
-    })(m1 = exports.m1 || (exports.m1 = {}));
+    })(exports.m1 || (exports.m1 = {}));
+    var m1 = exports.m1;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsMultipleTimesMultipleImport/amd/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsMultipleTimesMultipleImport/node/useModule.js [hunk 8]
```diff
@@ -11,7 +10,8 @@ var m1;
     var x3 = m4.x;
     var d3 = m4.d;
     var f3 = m4.foo();
-})(m1 = exports.m1 || (exports.m1 = {}));
+})(exports.m1 || (exports.m1 = {}));
+var m1 = exports.m1;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsMultipleTimesMultipleImport/node/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsSimpleImport/amd/useModule.js [hunk 9]
```diff
@@ -11,5 +10,6 @@ define(["require", "exports", "m4"], function (require, exports, m4) {
         var x3 = m4.x;
         var d3 = m4.d;
         var f3 = m4.foo();
-    })(m1 = exports.m1 || (exports.m1 = {}));
+    })(exports.m1 || (exports.m1 = {}));
+    var m1 = exports.m1;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsSimpleImport/amd/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/declarationsSimpleImport/node/useModule.js [hunk 10]
```diff
@@ -11,4 +10,5 @@ var m1;
     var x3 = m4.x;
     var d3 = m4.d;
     var f3 = m4.foo();
-})(m1 = exports.m1 || (exports.m1 = {}));
+})(exports.m1 || (exports.m1 = {}));
+var m1 = exports.m1;
```
Comment (rbuckton) [inline tests/baselines/reference/project/declarationsSimpleImport/node/useModule.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/typeResolution.js [hunk 11]
```diff
@@ -260,7 +259,8 @@ define(["require", "exports"], function (require, exports) {
             }());
             NotExportedModule.ClassA = ClassA;
         })(NotExportedModule || (NotExportedModule = {}));
-    })(TopLevelModule1 = exports.TopLevelModule1 || (exports.TopLevelModule1 = {}));
+    })(exports.TopLevelModule1 || (exports.TopLevelModule1 = {}));
+    var TopLevelModule1 = exports.TopLevelModule1;
```
Comment (rbuckton) [inline tests/baselines/reference/typeResolution.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/withExportDecl.js [hunk 12]
```diff
@@ -89,13 +89,13 @@ define(["require", "exports"], function (require, exports) {
...
     })(m1 || (m1 = {}));
-    var m3;
     (function (m3) {
         function foo() {
             return m1.foo();
         }
         m3.foo = foo;
-    })(m3 = exports.m3 || (exports.m3 = {}));
+    })(exports.m3 || (exports.m3 = {}));
+    var m3 = exports.m3;
```
Comment (rbuckton) [inline tests/baselines/reference/withExportDecl.js]: Don't accept this baseline. I have a fix for this.

File: tests/baselines/reference/project/baseline3/amd/nestedModule.js (hunk 1 above)
Comment (yuit) [inline tests/baselines/reference/project/baseline3/amd/nestedModule.js]: ok sound good.

**Comment by yuit** [comment]
@rbuckton  @vladima  Do other files look good (ie. the one not mark by Ron?) I will go ahead and remove the incorrect one

**Comment by rbuckton** [comment]
:+1:

**Pull request merged**